### PR TITLE
docs: Add document about using different format in frontmatter

### LINKED ISSUE #213: Add doc about the support front matter format
We use https://www.npmjs.com/package/gray-matter so we support multiple format already.

Ref https://github.com/MoOx/statinamic/pull/212/files#r54341605.

### PR DESCRIPTION
**Pull request by thangngoc89**
Close #213

### CODE REVIEW

**Comment by thangngoc89** [comment]
Craps. Wrong branch.

File: docs/content/docs/faq.md [hunk 1]
```diff
@@ -40,3 +40,20 @@ cover this case.
 -- import { Link } from "react-router"
 ++ import { Link } from "statinamic/lib/Link"
 ```
+
+### Can I use different font matter format in markdown files ?
+
+Yes. Statinamic supports YAML, JSON, TOML out of the box. You can change the language by
+specify it after the first delimiter (eg: ---)
```
Comment (ben-eb) [inline docs/content/docs/faq.md]: *specifying*

File: docs/content/docs/faq.md [hunk 2]
```diff
@@ -40,3 +40,20 @@ cover this case.
...
+
+```markdown
+---json
+{
+  "title": "Statinamic is awesome"
+}
+---
+```
+
+For more information, please refer [gray-matter's documentation](https://www.npmjs.com/package/gray-matter)
```
Comment (ben-eb) [inline docs/content/docs/faq.md]: *refer to*

File: docs/content/docs/faq.md (hunk 1 above)
Comment (thangngoc89) [inline docs/content/docs/faq.md]: :+1: thanks

File: docs/content/docs/faq.md (hunk 2 above)
Comment (thangngoc89) [inline docs/content/docs/faq.md]: :+1: thanks

**Comment by MoOx** [comment]
This should not be in faq. Next time wait for a peer review :)

**Comment by ben-eb** [comment]
Also I missed 'font matter' in the title, should be 'front matter' instead.

**Comment by thangngoc89** [comment]
:+1: Thanks @MoOx @ben-eb

**Pull request merged**